src/components/VideoPlayer/BaseVideoPlayer.js

```diff
@@ -96,20 +94,24 @@ function BaseVideoPlayer({
 
     const handlePlaybackStatusUpdate = useCallback(
         (e) => {
-            if (shouldReplayVideo(e, isPlaying, duration, position)) {
+            const isVideoPlaying = e.isPlaying || false;
+            const currentDuration = e.durationMillis || videoDuration * 1000;
+            const currentPositon = e.positionMillis || 0;
+
+            if (shouldReplayVideo(e, isVideoPlaying, currentDuration, currentPositon)) {
```

Comment:
This change of replacing `isPlaying` caused a bug https://github.com/Expensify/App/issues/37249